### ISSUE TITLE
Hyperedge Graphics

### ISSUE THREAD

**Issue by Zaphyz** [WolframInstitute/Hypergraph#1]
For early versions of hypergraph visualization poligon representations of hyperedges are fine but the elastic fabric representation is particularly useful to display hyperedges of different orders on the same set of vertices (particularly distinguishing order 2 from order >2). Since the graphical style is of some kind of rubbery matterial, that representation is also quite convenient for non-simple hypergraphs, where several hyperedges one the same set of vertices can be represented as bent elastic surfaces (like blown by the wind in different directions and intensities). For order 1 edges I would go for solid ball representations. In general, we should aim to use transparency cleverly.

In summary: 1-edges are represented by solid (3D) balls, 2-edges are represented by lines (1D) and >2-edges are represented by surfaces (2D, minimal under some constraints).
